custom/ewsghana/alerts/alerts.py

```diff
@@ -66,7 +66,7 @@ def on_going_process_user(user, test=False):
             send_sms_to_verified_number(verified_number, message)
         else:
             send_test_message(verified_number, message)
-        if user.email:
+        if user.email and verified_number.backend_id == 'MOBILE_BACKEND_TWILIO':
```

Comment:
This general approach looks right to me (@czue feel free to weight in), but if we take this approach, wouldn't it be better to create a function called can_receive_email that behaves like this:

    def can_receive_email(user, verified_number)
        return True if user.email and verified_number.backend_id and verified_number.backend_id != 'MOBILE_BACKEND_TEST' else False

This way, we define an 'ignored user' to be a user who does not have a backend_id set or has the test backend_id set, and anyone who has an actual backend_id set will receive both sms and email content.  Then, when the testing is over you can just change it to

    def can_receive_email(user, verified_number)
        return True if user.email else False